pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Describe your changes
+
+Please include a clear overview of what has changed so that reviewers can quickly understand what this pull request does. If this PR contains changes to multiple files, provide guidance to reviewers about the order in which to review the files.
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## What kind of feedback do you need?
+
+E.g. Do you need a quick look or a thorough assessment?
+
+## Checklist:
+
+- [ ] I have performed a self-review of my code
+- [ ] I have recommended where to start and how to proceed with the review
+
```

Comment:
Do we need to include the 'reviewer nomination' in this space, or is it somewhere further in the process?